mps_youtube/config.py

```diff
@@ -67,6 +67,9 @@ def set(self, value):
 
         # handle known player not set
 
+        # sadly, allowed_values is faulty - assumes copy() is always
+        # accessible, causing exception on any string not among allowed_values
```

Comment:
Sure, please check  what happens if user didn't read `help config` careful, or made a typo like this
`set save_history search_onl` (notice missing 'y'). Exception is generated like this:

```
> set save_history search_onl
Traceback (most recent call last):
  File "./mpsyt", line 3, in <module>
    mps_youtube.main.main()
  File "/tmp/mps-youtube/mps_youtube/main.py", line 3819, in main
    if matchfunction(k, v, userinput):
  File "/tmp/mps-youtube/mps_youtube/main.py", line 3716, in matchfunction
    func(*matches)
  File "/tmp/mps-youtube/mps_youtube/main.py", line 471, in setconfig
    message = Config[key.upper()].set(val)
  File "/tmp/mps-youtube/mps_youtube/config.py", line 75, in set
    allowed_values = self.allowed_values.copy()
AttributeError: 'list' object has no attribute 'copy'
```

also happens when user tries to change user_order and other parameters that accept multiple values.